core/entities/entity.py

```diff
@@ -80,12 +80,22 @@ def generate_tags(self):
         return []
 
     def info(self):
+        """Object info.
+        
+        When there is no Flask context, url and human_url are not returned and the
+        object id is returned instead.
+        """
         i = {
             "name": self.name,
             "description": self.description,
             "tags": self.tags
         }
-        i['url'] = url_for("api.Entity:post", id=str(self.id), _external=True)
-        i['human_url'] = url_for(
-            "frontend.EntityView:get", id=str(self.id), _external=True)
+        try:
+            i['url'] = url_for("api.Entity:post", id=str(self.id), _external=True)
+            i['human_url'] = url_for(
+                "frontend.EntityView:get", id=str(self.id), _external=True)
+        except RuntimeError:
+            i['id'] = str(self.id)
```

Comment:
Feel free, I thought it was important to return the id if the URL wasn't present. It was an unused code path, since the code previously exited with an exception.